cli/core/messages.go

```diff
@@ -0,0 +1,37 @@
+package core
+
+import (
+	"fmt"
+)
+
+func plural(word string, amount int) string {
+	if amount > 1 {
+		return word + "s"
+	}
+	return word
+}
+
+func SubmitCheckpointProofConsent(numTransactions int) string {
+	return fmt.Sprintf(`This will start a new checkpoint on your eigenpod.
+
+	Once started, you MUST complete this checkpoint. This may entail submitting %d or more transactions.
```

Comment:
> This may entail submitting %d or more transactions.

I would be more concrete, just like "This will require %d transactions"

One additional question is whether to include the call to `startCheckpoint` in `numTransactions`. It might be confusing if you don't.

Reply:
So actually that would mean all checkpoints require 2 transactions, making the `plural` usage irrelevant -- it will always say `"times"`

Reply:
yep

Reply:
updated language